build.sbt

```diff
@@ -116,8 +118,14 @@ lazy val server = project
   .settings(serverSettings)
 
 //////////////////////////////
-//// HEALTHCHECK  EXAMPLE ////
+//// HEALTHCHECK   ////
 //////////////////////////////
+lazy val `health-check-unary` = project
+  .in(file("modules/health-check-unary"))
+  .dependsOn(channel)
+  .dependsOn(server)
```

Comment:
I think you don't need `server`

Reply:
That's right,  I need it just on the examples